crypto/rand/rand_lib.c

```diff
@@ -172,7 +170,7 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
             if (RAND_DRBG_generate(drbg->parent,
                                    buffer, bytes_needed,
                                    prediction_resistance,
-                                   NULL, 0) != 0)
+                                   (unsigned char *)&drbg, sizeof(drbg)) != 0)
```

Comment:
A comment explaining this additional data would be useful

Reply:
> A comment explaining this additional data would be useful

Done, see https://github.com/openssl/openssl/pull/9802/commits/448a5b058d64bfc430d8be860ce38c1d88ad8a1e#diff-baca3d0b945f277b98ef9b654387b6bfR164-R166.